packages/story-editor/src/app/history/historyProvider.tsx

```diff
@@ -17,14 +17,16 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import {
   useCallback,
   useEffect,
   useState,
   useRef,
 } from '@googleforcreators/react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment -- @todo Let's fix as part of #12656.
+// @ts-ignore
 import { useGlobalKeyDownEffect } from '@googleforcreators/design-system';
```

Comment:
@barklund / @swissspidy `useGlobalKeyDownEffect` is part of a larger file which has a bunch of related functions. Is there a better alternative without just doing TS ignore for now and without having to convert all those?

I could alternatively leave this ignore in for now and create a new issue for converting the `keyboard` folder of the design-system which would also remove this ignore here.

Thoughts?

Reply:
How much work would it be to convert it? It seems worth doing

Reply:
You mean worth doing it within this PR instead of separately? Probably a 2-pointer, at the first glance doesn't look like there's anything overly complex. I'll just add it within this PR then.